{"name": "test_login_fail[123-123]", "status": "broken", "statusDetails": {"message": "AttributeError: 'NoneType' object has no attribute 'clear'", "trace": "self = <pytest1.test_login.test_login.Test_login object at 0x00000257F6A30490>\nname = '123', pwd = '123'\n\n    @allure.story('不正确的用户名和密码进行登录')\n    @allure.severity(allure.severity_level.CRITICAL)\n    @pytest.mark.parametrize(\"name,pwd\", [('zhaoyajing@yundun.com','123')#正确用户名错误的密码\n        ,('123','4726586z.')#错误用户名正确的密码\n        ,('123','123')#错误用户名错误的密码\n        ,('','4726586z.')#不填用户名，正确的密码\n        ,('zhaoyajing@yundun.com','')#正确的用户名，不填密码\n                                          ])\n    def test_login_fail(self,name,pwd):\n        self.page.open('https://www.yundun.com/login')\n        sleep(2)\n>       self.page.type_username(name)\n\ntest_login.py:45: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <page.login_page.Login_page object at 0x00000257F71512E0>, value = '123'\n\n    @allure.step('输入用户名:{1}')\n    def type_username(self,value):\n        ele = self.find_element('id','username')\n>       ele.clear()\nE       AttributeError: 'NoneType' object has no attribute 'clear'\n\n..\\page\\login_page.py:14: AttributeError"}, "steps": [{"name": "打开页面:'https://www.yundun.com/login'", "status": "passed", "parameters": [{"name": "url", "value": "'https://www.yundun.com/login'"}], "start": 1607583122492, "stop": 1607583124775}, {"name": "输入用户名:'123'", "status": "broken", "statusDetails": {"message": "AttributeError: 'NoneType' object has no attribute 'clear'\n", "trace": "  File \"D:\\software\\python\\lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"D:\\workspace\\pytest1\\page\\login_page.py\", line 14, in type_username\n    ele.clear()\n"}, "steps": [{"name": "查找元素:类型-'id'，定位：'username'", "status": "passed", "attachments": [{"name": "获取单个元素失败", "source": "3feec910-3731-44dd-bacb-7eeeed490a1d-attachment.png", "type": "image/png"}], "parameters": [{"name": "type", "value": "'id'"}, {"name": "path", "value": "'username'"}, {"name": "time_wait", "value": "2"}], "start": 1607583126785, "stop": 1607583129148}], "parameters": [{"name": "value", "value": "'123'"}], "start": 1607583126785, "stop": 1607583129148}], "parameters": [{"name": "name", "value": "'123'"}, {"name": "pwd", "value": "'123'"}], "start": 1607583122491, "stop": 1607583129148, "uuid": "d3487392-3e66-4b90-bd1a-b2cfab2c7209", "historyId": "d0ae12fe11b757d5abf28e7b602fd961", "testCaseId": "11b0076a5e8bcdc60388920dfd428cb1", "fullName": "test_login.test_login.Test_login#test_login_fail", "labels": [{"name": "feature", "value": "登录"}, {"name": "severity", "value": "critical"}, {"name": "story", "value": "不正确的用户名和密码进行登录"}, {"name": "tag", "value": "page"}, {"name": "parentSuite", "value": "test_login"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_login"}, {"name": "host", "value": "LAPTOP-HRAELMRK"}, {"name": "thread", "value": "20932-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login.test_login"}]}